crypto/tls/test/tls_integration_test.go

```diff
@@ -55,21 +55,21 @@ func (h *grpcHealthCheck) Watch(_ *grpc_health_v1.HealthCheckRequest, _ grpc_hea
 	return status.Error(codes.Unimplemented, "Watching is not supported")
 }
 
-func getLocalHostPort() (int, error) {
+func getLocalHostPort() (*net.TCPAddr, error) {
```

Comment:
Should we rename this method now that it's no longer returning only the port? `getLocalHostAddr` perhaps?